aggsender/db/aggsender_db_storage.go

```diff
@@ -179,10 +211,38 @@ func getCertificateByHeight(db dbtypes.Querier,
 		"SELECT * FROM certificate_info WHERE height = $1;", height); err != nil {
 		return nil, getSelectQueryError(height, err)
 	}
-
 	return &certificateInfo, nil
 }
 
+// getCertificatesByHeight returns a certificates in the required table
+func getCertificatesByHeight(db dbtypes.Querier, table tableName,
```

Comment:
Consider making this a method, so then you can remove the `db` param and reuse the `db` reference from the instance. WDYT?
```suggestion
func (a *AggSenderSQLStorage) getCertificatesByHeight(table tableName,
```

Reply:
I did in that way because pre-existing helper function were done in that way https://github.com/agglayer/aggkit/blob/9d03f6d5ce83086a5f7c2c24a934bf1b42a59486/aggsender/db/aggsender_db_storage.go#L207
I agree, that I prefer doing as a member of the object, but that implies refactor several function to match the same criteria. Also you can't save the param `db dbtypes.Querier` because is required to pass the transaction
WDYT?